mmv1/third_party/terraform/tests/resource_bigtable_instance_test.go

```diff
@@ -478,6 +487,51 @@ resource "google_bigtable_instance" "instance" {
 `, instanceName, instanceName, numNodes, instanceName, numNodes, instanceName, numNodes)
 }
 
+// Add two clusters after testAccBigtableInstance_clusterModified.
+func testAccBigtableInstance_clusterModifiedAgain(instanceName string, numNodes int) string {
+	return fmt.Sprintf(`
+resource "google_bigtable_instance" "instance" {
+  name = "%s"
+  cluster {
+    cluster_id   = "%s-c"
+    zone         = "us-central1-c"
+    num_nodes    = %d
+    storage_type = "HDD"
+  }
+  cluster {
+    cluster_id   = "%s-a"
+    zone         = "us-central1-a"
+    num_nodes    = %d
+    storage_type = "HDD"
+  }
+  cluster {
+    cluster_id   = "%s-b"
+    zone         = "us-central1-b"
+    num_nodes    = %d
+    storage_type = "HDD"
+  }
+  cluster {
+    cluster_id   = "%s-y"
+    zone         = "us-central1-y"
+    num_nodes    = %d
+    storage_type = "HDD"
+  }
+  cluster {
+    cluster_id   = "%s-x"
+    zone         = "us-central1-x"
+    num_nodes    = %d
+    storage_type = "HDD"
+  }
+
```

Comment:
This test fails to run... these zones `-y` and `-x` do not exist

Reply:
Sorry for delay. Would it be an intended test if changed the existed zones...?

Reply:
yes, this test cannot run as these zones are not real. Thus the test cannot run.